src/keyfile.ts

```diff
@@ -20,6 +20,7 @@ function slugify(text: string) {
 
 export async function writeAPIKey(key: string): Promise<void> {
   const keyPath = join(envHOMEDIR(), KEY_FILE);
+  log.debug("Key path", keyPath);
```

Comment:
@oganexon Should this still be here?

Reply:
@t8 Yes, by default it is not displayed 
You have to use the `--debug` option to the message